components/org.wso2.carbon.identity.oauth.uma.resource.endpoint/src/gen/java/org/wso2/carbon/identity/oauth/uma/resource/endpoint/dto/ResourceDetailsDTO.java

```diff
@@ -49,12 +49,12 @@
      **/
     @ApiModelProperty(required = true, value = "An array of strings indicating the available scopes for this resource.\n")
     @JsonProperty("resource_scope")
-    public List<String> getResource_scopes() {
+    public List<String> getResourceScopes() {
         return resource_scopes;
```

Comment:
Change these values also to camel case

Reply:
In UMA specification it's given as resource_scopes

Reply:
Isn't this an auto generated code ? If UMA spec specifies to use Resource_scope we can go ahead with getResource_scopes. Otherwise we have the overhead of doing these changes manually in all auto generated java code whenever DTOs are re-generated. This is an overhead and since this is autogenerated we can disregard standards.

Reply:
In a review, we were discussed to change getters and setters in to camel Case.

Reply:
Agree with @hasinthaindrajee. Keep the auto generated code as it is. Otherwise it will be a overhead for future development.